mongo/index_view.go

```diff
@@ -276,7 +276,7 @@ func (iv IndexView) CreateMany(ctx context.Context, models []IndexModel, opts ..
 		Session(sess).WriteConcern(wc).ClusterClock(iv.coll.client.clock).
 		Database(iv.coll.db.name).Collection(iv.coll.name).CommandMonitor(iv.coll.client.monitor).
 		Deployment(iv.coll.client.deployment).ServerSelector(selector).ServerAPI(iv.coll.client.serverAPI).
-		Timeout(iv.coll.client.timeout).MaxTime(option.MaxTime)
+		Timeout(iv.coll.client.timeout).MaxTime(option.MaxTime).Crypt(iv.coll.client.cryptFLE)
```

Comment:
Suggest applying a similar change to the `List` and `drop` methods. Those methods also do not pass the `Crypt` methods on the created operations. I expect the operations are not processed for automatic encryption.

Consider filing a separate GODRIVER Bug ticket to track the fix. I expect this bug fix may be desirable to backport into the 1.x driver.

Drivers are expected to process `createIndexes` for automatic encryption. The `createIndexes` command may reference encrypted fields. See https://github.com/mongodb/specifications/pull/1227:

> The "create" and "collMod" commands may contain sensitive data in `validators`:
> 
> ```js
> {
>     "create": "encryptedCollection",
>     "validator": {
>         // "a", "b", and "c" are sensitive.
>         "encryptedField": { "$in": [ "a", "b", "c" ]}
>     }
> }
> ```

Reply:
Created GODRIVER-3038 , thanks!